classes/module/Module.php

```diff
@@ -271,6 +271,9 @@ public static function processDeferedClearCache()
      */
     public function __construct($name = null, Context $context = null)
     {
+        if (is_null($this->ps_versions_compliancy))
+            $this->ps_versions_compliancy = [];
```

Comment:
```suggestion
        if (is_null($this->ps_versions_compliancy)) {
            $this->ps_versions_compliancy = [];
        }
```

Reply:
Never use `is_null`, it's really really really slow compared to `!== null`